.travis.yml

```diff
@@ -1,25 +1,24 @@
 addons:
-  firefox: "49.0.2"
+  firefox: "54.0.1"
```

Comment:
this is the browser I'm using right now

I fiddled with the browser version and gecko driver version a _lot_, but this seems to work reliably nicely on CI